assets/dev/scss/editor/panel/_controls.scss

```
@@ -504,6 +504,8 @@ $devices: 'desktop', 'tablet', 'mobile';
 
 @import "controls/button";
 
+@import "controls/code";
+
 @import "controls/choose";
```

Order alphabetically.
```suggestion
@import "controls/choose";

@import "controls/code";
```

---

Done.